redisinsight/api/test/helpers/server.ts

```diff
@@ -41,6 +42,7 @@ export const getServer = async () => {
     app.use(bodyParser.json({ limit: '512mb' }));
     app.use(bodyParser.urlencoded({ limit: '512mb', extended: true }));
     app.use('/static', express.static(serverConfig.get('dir_path').staticDir))
+    app.useWebSocketAdapter(new SessionMetadataAdapter(app));
```

Comment:
@arthosofteq this is the line of code I had to add for the BE integration tests to pass